rudder-web/src/main/scala/com/normation/rudder/web/snippet/administration/PropertiesManagement.scala

```diff
@@ -555,10 +555,15 @@ class PropertiesManagement extends DispatchSnippet with Loggable {
               def radioHtml(label:String) : NodeSeq = {
                 val inputId = label+"-id"
                 val ajaxCall = SHtml.ajaxCall(Str(""), _ => setRelaySyncMethodJs(label))._2.toJsCmd
+                val checked = relaySyncMethod.value == label
+                var inputCheck = checked match {
```

Comment:
That should be a val, you should never do a var !

Reply:
val inputCheck = if (relaySyncMethod.value == label) {
  <input id={inputId} type="radio" name="relaySync" onclick={ajaxCall} checked=""/>
} else {
  <input id={inputId} type="radio" name="relaySync" onclick={ajaxCall}/>
}